controllers/replace_failed_process_groups_test.go

```diff
@@ -139,7 +136,7 @@ var _ = Describe("replace_failed_process_groups", func() {
 			targetPodProcessGroupID = internal.GetProcessGroupIDFromMeta(cluster, podOnTaintedNode.ObjectMeta)
 
 			// Call validateProcessGroups to set processGroupStatus to tainted condition
-			processGroupsStatus, err := validateProcessGroups(ctx.TODO(), clusterReconciler, cluster, &cluster.Status, processMap, configMap, allPods, allPvcs, logger)
+			processGroupsStatus, err := validateProcessGroups(ctx.TODO(), clusterReconciler, cluster, &cluster.Status, processMap, configMap, allPvcs, logger)
```

Comment:
Why using "allPvcs" instead of "allPods"?

Reply:
Same question for the changes below.

Reply:
This is only needed until we fetch the pvcs directly from the cache similar to what we did in this PR for Pods. We still validate all Pods but the Pod is now fetched from the controller-runtime cache, so we don't have to pass down the Pod map. Created: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/1638 as follow up.